contrib/util/ccda_import/import_ccda.php

```diff
@@ -47,6 +47,9 @@
     die;
 }
 
+// get around a large ccda data array
+ini_set("memory_limit", -1);
```

Comment:
![Uploading 2DV.gif…]()

Reply:
![mrw-im-at-a-new-barber-shop-and-immediately-get-called-ahead-of-everybody-waiting-unknown-to--barber-159555](https://user-images.githubusercontent.com/5618372/125011920-52171680-e037-11eb-8c65-ac56ad9b9b2d.gif)